packages/delivery-service/src/validation/notification/regexValidation.ts

```diff
@@ -17,7 +17,13 @@ export const checkRegexPattern = (
                 .match(EMAIL_REGEX);
             return patternCheck;
         case NotificationChannelType.PUSH:
-            return typeof recipientValue !== 'string' ? true : false;
+            const data = JSON.parse(recipientValue);
```

Comment:
looks way better. Please get rid of the doubled negation to make it perfect.
return data.endpoint &&
                data.keys &&
                data.keys.p256dh &&
                data.keys.auth